README.md

```diff
@@ -1,125 +1,79 @@
-The example fits an SDE to data, all the while regularizing it to be like an [Ornstein-Uhlenbeck](https://en.wikipedia.org/wiki/Ornstein%E2%80%93Uhlenbeck_process) prior process.
-The model can be loosely viewed as a [variational autoencoder](https://en.wikipedia.org/wiki/Autoencoder#Variational_autoencoder_(VAE)) with its prior and approximate posterior being SDEs.
-Note, the example contains many simplifications and is meant to demonstrate basic usage regarding gradient computation.
-
-To run the latent SDE example, execute the following command from the root folder of this repo:
-```shell script
-python3 -m examples.latent_sde --train-dir ${TRAIN_DIR}
-```
-Once in a while, the program writes a figure to the path specified by `TRAIN_DIR`.
-Training should stabilize after 500 iterations with the default hyperparameters.
-
-## Usage
-The central functions of interest are `sdeint` and `sdeint_adjoint`. They can be imported as follows:
-```Python
-from torchsde import sdeint, sdeint_adjoint
-```
-
-### Integrating SDEs
-SDEs are defined with two vector fields, named respectively the *drift* and *diffusion* functions.
-The drift is analogous to the vector field in ODEs, whereas the diffusion controls how the [Brownian motion](https://en.wikipedia.org/wiki/Brownian_motion) affects the system.
-
-By default, solvers in the codebase integrate SDEs that are `torch.nn.Module` objects with the drift function `f` and diffusion function `g`.
-For instance, the [Geometric Brownian motion](https://en.wikipedia.org/wiki/Geometric_Brownian_motion) can be integrated as follows:
-```Python
+### Quick example
+```python
 import torch
-from torchsde import sdeint
+import torchsde
 
+batch_size, state_size, brownian_size = 32, 3, 2
+t_size = 20
 
 class SDE(torch.nn.Module):
+    noise_type = 'general'
+    sde_type = 'ito'
 
-    def __init__(self, mu, sigma):
+    def __init__(self):
         super().__init__()
-        self.noise_type="diagonal"
-        self.sde_type = "ito"
-
-        self.mu = mu
-        self.sigma = sigma
+        self.mu = torch.nn.Linear(state_size, 
+                                  state_size)
+        self.sigma = torch.nn.Linear(state_size, 
+                                     state_size * brownian_size)
 
     def f(self, t, y):
-        return self.mu * y
+        return self.mu(y)  # shape (batch_size, state_size)
 
     def g(self, t, y):
-        return self.sigma * y
+        return self.sigma(y).view(batch_size, 
+                                  state_size, 
+                                  brownian_size)
+
+sde = SDE()
+y0 = torch.full((batch_size, state_size), 0.1)
+ts = torch.linspace(0, 1, t_size)
+# Initial state y0, the SDE is solved over the interval [ts[0], ts[-1]].
+# ys will have shape (t_size, batch_size, state_size)
+ys = torchsde.sdeint(sde, y0, ts)
+```
+
+### Notebook
```

Comment:
I think it might be worthwhile to put Notebook and LatentSDE under a section titled `examples`. You can then put your SDE-GAN model also under this section later as a subsection/paragraph, if you feel like it.

Reply:
They are already under a section titled examples?

I would like to add an SDE-GAN example as well.

Reply:
Ah sorry, you're right.